virtualizarr/xarray.py

```diff
@@ -18,6 +18,7 @@
 from xarray.core.variable import IndexVariable
 
 import virtualizarr.kerchunk as kerchunk
+from virtualizarr.dmrpp import DMRParser
```

Comment:
In #87 Sean is adding a `readers` submodule - let's follow that pattern here and have `readers/dmrpp`.